### PR TITLE
Adding Dockerfile with Unubtu 18:04 and Python3

### PR DESCRIPTION
**Pull request by rliberma**
#176 

Adding Dockerfile to the project root folder to run Ubuntu 18.04 and Python3.

### CODE REVIEW

**Review by IlanZuckerman — DISMISSED** [review]
LGTM

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,6 @@
+FROM ubuntu:18.04
+MAINTAINER Canonical
```
Comment (lioramilbaum) [inline Dockerfile:2]: Who is Canonical?

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,6 @@
...
+
+RUN apt-get update -y
+RUN apt-get install -y python3-pip python3-setuptools mysql-server mysql-client libsystemd-dev
```
Comment (lioramilbaum) [inline Dockerfile:5]: Why mysql-server and mysql-client are required?

File: Dockerfile [hunk 3]
```diff
@@ -0,0 +1,6 @@
...
+
```
Comment (lioramilbaum) [inline Dockerfile:6]: extra line

**Review by lioramilbaum — CHANGES_REQUESTED** [review]

File: Dockerfile (hunk 1 above)
Comment (rliberma) [reply Dockerfile:2]: They are the Ubuntu official maintainer.

File: Dockerfile (hunk 2 above)
Comment (rliberma) [reply Dockerfile:5]: This is all the packages we are installing today to our application. I can remove them if you think we will not use them. ATM we are not using them.

File: Dockerfile (hunk 1 above)
Comment (lioramilbaum) [reply Dockerfile:2]: He is not relevant for a layered image. You should put your name.

File: Dockerfile (hunk 2 above)
Comment (lioramilbaum) [reply Dockerfile:5]: Then remove whatever is not used.

File: Dockerfile [hunk 4]
```diff
@@ -0,0 +1,3 @@
+FROM ubuntu:18.04
+RUN apt-get update -y
+RUN apt-get install -y python3-pip python3-setuptools libsystemd-dev
```
Comment (lioramilbaum) [inline Dockerfile:3]: Merge the RUN commands into one chained RUN instruction.
You can find the reasoning for that in the following KB:
https://stackoverflow.com/questions/39223249/multiple-run-vs-single-chained-run-in-dockerfile-what-is-better

**Review by lioramilbaum — CHANGES_REQUESTED** [review]

**Review by IlanZuckerman — APPROVED** [review]
LGTM

**Review by lioramilbaum — APPROVED** [review]

**Pull request merged**